core/src/main/java/jenkins/security/UpdateSiteWarningsMonitor.java

```diff
@@ -155,9 +155,9 @@ public HttpResponse doForward(@QueryParameter String fix, @QueryParameter String
     }
 
     /**
-     * Returns true iff there are applicable but ignored (i.e. hidden) warnings.
+     * Returns true if there are applicable but ignored (i.e. hidden) warnings.
```

Comment:
These are not typos.

*iff* is shorthand for *if and only if*, see e.g. http://www.dictionary.com/browse/iff